### PR TITLE
Passenger | Add endpoint to create paypal client token

### PR DESCRIPTION
**Pull request by phylor**
Adds the endpoint to create a paypal client token for Logpay.

### CODE REVIEW

**Comment by phylor** [comment]
@rmehner This is finally read again to be reviewed!

File: lib/ioki/apis/passenger_api.rb [hunk 1]
```diff
@@ -138,11 +138,22 @@ class PassengerApi
...
       Endpoints::Index.new(
         :payment_methods,
         base_path:   [API_BASE_PATH],
         model_class: Ioki::Model::Passenger::PaymentMethod
       ),
+      Endpoints::Create.new(
+        :payment_method,
+        base_path:   [API_BASE_PATH],
+        model_class: Ioki::Model::Passenger::PaymentMethod
+      ),
```
Comment (rmehner) [inline lib/ioki/apis/passenger_api.rb:156]: I'd expect a test for this in https://github.com/ioki-mobility/ioki-ruby/blob/a6ca93f8f5f222b451eb9fd0b083aa08844c418d/spec/ioki/passenger_api_spec.rb#L308

**Review by rmehner — APPROVED** [review]
Looks good, misses one spec :)

File: lib/ioki/apis/passenger_api.rb (hunk 1 above)
Comment (phylor) [reply lib/ioki/apis/passenger_api.rb:156]: Added!

**Pull request merged**